apps/server/src/modules/provisioning/strategy/oidc/oidc.strategy.ts

```diff
@@ -31,13 +31,15 @@ export abstract class OidcProvisioningStrategy extends ProvisioningStrategy {
 			);
 
 			if (data.externalGroups) {
+				let groups: ExternalGroupDto[] = data.externalGroups;
+
+				if (Configuration.get('FEATURE_VIEW_PROVISIONING_OPTIONS_ENABLED')) {
```

Comment:
Please use a config module and dont use the configuration directly. There were a decision by the arc group about that

I know you didnt introduced the name of the feature but i dont like the VIEW. It doesnt handle only the view since we are using it in the backend.

Maybe something like FEATURE_PROVISIONING_OPTIONS_ENABLED.